captchasrv/mkquizword.pl

```diff
@@ -22,13 +22,14 @@
 #   - English dictionary file, 1 word per line, see constant DictFn.
 # Returns: quizword string
 ###
+use File::Basename;
 
 sub mkquizworddict {
 
   ##########
   # Constant section - Personnalize here
-  ## Installation dir of this program - IMPORTANT
-  *InstallDir=\ "/home/mora/src/spamgourmet/captcha";
+  ## Installation dir of this program - Used to find data files in same dir as this scrupt
+  *InstallDir=\ dirname(__FILE__);;
```

Comment:
i.e., also here. Maybe factor out `InstallDir` to a common module?

Reply:
You don't want this symbol to go to a different file!

Reply:
Not even if it is in the same directory?